x-pack/plugins/enterprise_search/public/applications/app_search/components/credentials/credentials_list/credentials_list.test.tsx

```diff
@@ -120,24 +134,13 @@ describe('Credentials', () => {
         hidePerPageOptions: true,
       });
     });
-
-    it('will default pagination values if `page` is not available', () => {
-      setMockValues({ ...values, meta: {} });
-      const wrapper = shallow(<CredentialsList />);
-      const { pagination } = wrapper.find(EuiBasicTable).props();
-      expect(pagination).toEqual({
-        pageIndex: 0,
-        pageSize: 0,
-        totalItemCount: 0,
-        hidePerPageOptions: true,
-      });
-    });
```

Comment:
I removed this test as it no longer seems necessary with the `DEFAULT_META` change